initializers/db.go

```diff
@@ -0,0 +1,38 @@
+package initializers
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"tiny-site-backend/models"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+)
+
+var DB *gorm.DB
+
+func ConnectDB(config *Config) {
```

Comment:
here ConnectDB() function, where the DB.Exec() method is used to create the uuid-ossp extension. The DB.Exec() method is used to execute raw SQL queries, and it is important to sanitize the input to prevent SQL injection attacks.

Reply:
I have added parameterized query to create the uuid-ossp extension